### PR TITLE
[incubator/jaeger]Option to provide sampling strategies file

### PR DESCRIPTION
**Pull request by naseemkullah**
#### What this PR does / why we need it:
Allows users to pass along a sampling strategies file as per https://www.jaegertracing.io/docs/1.15/sampling/#collector-sampling-configuration

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/jaeger/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.15.1
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.13.4
+version: 0.14.0
```
Comment (pavelnikolov) [inline incubator/jaeger/Chart.yaml]: There is already another PR which is approved and has the same version.

**Review by pavelnikolov — APPROVED** [review]

File: incubator/jaeger/Chart.yaml (hunk 1 above)
Comment (naseemkullah) [reply incubator/jaeger/Chart.yaml]: Yes, making 1 logical change per PR now :)
Was not sure which would merge first, will update now, thanks!

**Comment by naseemkullah** [comment]
Needs another /lgtm @pavelnikolov, thanks!

**Pull request merged**